SEC/Securities/SecuritiesListings.rdf

```diff
@@ -147,7 +149,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>listed security</rdfs:label>
-		<skos:definition>registered security listed on one or more exchanges</skos:definition>
+		<skos:definition>registered security listed on some exchange</skos:definition>
```

Comment:
This looks like a minor wording change but has a major impact. It in effect replaces what was Listing and so needs several of the latter's properties moved over e.g. a link to the IssuedSecurity that is listed, and the currency